src/index.html

```diff
@@ -2,13 +2,54 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <div class="wrapper">
+      <header class="header">
+        <div class="header__container">
+          <a href="#" class="header__logo">
+            <img src="./images/logo.png" alt="Moyo Logo">
+          </a>
+          <nav class="header__menu menu">
+            <ul class="menu__list">
+              <li class="menu__item">
+                <a href="#" class="menu__link">Apple</a>
+              </li>
+              <li class="menu__item">
+                <a href="#" class="menu__link">Samsung</a>
+              </li>
+              <li class="menu__item">
+                <a href="#" class="menu__link">Smartphones</a>
+              </li>
+              <li class="menu__item">
+                <a href="#"
+                class="menu__link"
+                data-qa="hover"
+                >Laptops & Computers</a>
```

Comment:
Fix formatting everywhere

```suggestion
              <li class="menu__item">
                <a 
                  href="#"
                  class="menu__link"
                  data-qa="hover"
                >
                  Laptops & Computers
                </a>
```